src/site/changelog/single_release.rs

```diff
@@ -0,0 +1,89 @@
+use crate::errors::*;
+use crate::site::link;
+use crate::site::markdown::{self, SyntaxTheme};
+use axohtml::dom::UnsafeTextNode;
+use axohtml::elements::{a, section};
+use axohtml::html;
+use axohtml::{text, unsafe_text};
+use chrono::DateTime;
+
+use super::types::ReleasesApiResponse;
+
+pub fn build_single_release(
+    release: &ReleasesApiResponse,
+    syntax_theme: &SyntaxTheme,
+    version: &Option<String>,
```

Comment:
version should be required - and/or what is this version from? we may want to annotate it's origin since there's also a version in the api response right?

Reply:
Version is now required, I show an error if none is present and changed the name of the version param to be config_param, so we know where it comes from